docs/user_guide/context_actions.md

```diff
@@ -27,6 +27,10 @@ Sub-actions allow for copying and pasting the hovered units' loadout onto anothe
 Opens the Zeus garage on the hovered vehicle.
 Sub-actions allow for copying and pasting the hovered vehicle's customization onto another.
 
+# Editable Objects
+
+Allows Zeus to add or remove editable objects within the selected radius.
```

Comment:
```suggestion
Allows the Zeus to add or remove editable objects within the selected radius.
```

Reply:
I think no "the" is correct as there can be multiple, it is like this in other descriptions as well.

Reply:
I always say Zeuses for multiple.

Reply:
To me "Allow Zeus" reads smoother than "Allow the Zeus". But that might just be because I got used to it.